src/pages/api/event/[id]/repairers.ts

```diff
@@ -55,3 +58,52 @@ async function createRepairer(
     .status(200)
     .json({ message: "Successfully added volunteers to an event" });
 }
+
+async function getRepairers(req: NextApiRequest, res: NextApiResponse) {
+  const eventId = z.string().parse(req.query.id);
+  const result = await getEventRepairers(eventId as string);
+
+  res.status(200).json(result);
+}
+
+async function getEventRepairers(id: string) {
+  // get repairers from table EventRepairer
+  const eventRepairer = await prisma.eventRepairer.findMany({
+    where: { eventId: id }
+  });
+
+  if (eventRepairer.length === 0) {
+    return [];
+  }
+
+  const userIds = eventRepairer.map((repairer) => repairer.userId);
+
+  // get additional informations like firstName, lastName, email
+  const repairersUsers: Partial<Record<string, User>> =
+    await userService.getUserMapFromIds(userIds);
+
+  /* eslint-disable no-console */
+  if (Object.keys(repairersUsers).length !== userIds.length) {
```

Comment:
If you really want to add this error handling, it should be inside the getUserMapFromIds implementation